### ISSUE TITLE
FEATURE: Show focused outline when editing text

### ISSUE THREAD

**Issue by Sebobo** [neos/neos-ui#3853]
### Description

Currently no outline is shown when editing inline text when the mouse is not hovering the CKEditor field.

See also 

### Steps to Reproduce

1. Edit content and move mouse away

#### Expected behavior

![CleanShot 2024-09-19 at 10 52 57@2x](https://github.com/user-attachments/assets/e5d65677-a1e6-439e-800b-83cc56b99828)

#### Actual behavior

![CleanShot 2024-09-19 at 11 00 00@2x](https://github.com/user-attachments/assets/a329df98-0d50-44fe-a45e-4877eae3cc8d)

### Affected Versions

Neos: 8.3
UI: 8.3

**Comment by Sebobo** [comment]
Possible solution something like this (with a different color of course):

```css
.ck-focused {
    outline: 1px dashed red;
    outline-offset: 5px;
}
```

**Comment by mhsdesign** [comment]
fixed by https://github.com/neos/neos-ui/commit/0f544aa3fa34429a7e56ddd69a39a4fe9e830abf

The dottet border known during hover will now be kept during editing a node.

<img width="545" height="242" alt="Image" src="https://github.com/user-attachments/assets/069811d7-c2a3-43bf-b196-2de74a8b9616" />

There is one exception: When the node only has one html element and that is the ckeditor editable we cannot show two outlines as there is just one element. For these simple editables we omit the dotted outline and instead only show the solid blue focused node outline.

<img width="547" height="185" alt="Image" src="https://github.com/user-attachments/assets/a18f7885-3af4-469c-ad85-85218e2388ce" />
